CONTRIBUTING.md

```diff
@@ -110,4 +116,20 @@ Project maintainers who do not follow or enforce the Code of Conduct in good fai
 
 
 ### Attribution
-This Code of Conduct is adapted from the Contributor Covenant, version 1.4, available at http://contributor-covenant.org/version/1/4
\ No newline at end of file
+This Code of Conduct is adapted from the Contributor Covenant, version 1.4, available at http://contributor-covenant.org/version/1/4
+
+## Getting started
+
+### Prerequisites
+ - Install [Vagrant](https://www.vagrantup.com/downloads).
+ - Install [Virtual Box](https://www.virtualbox.org/wiki/Downloads).
+ - Install [Python 3.9](https://www.python.org/downloads/).
+
+### Getting a copy of the repository 
+1. Go to the [Xpense](https://github.com/redhat-beyond/Xpense) GitHub repository.
+2.  Fork the repository and create a separate branch regarding your contribution.
+3. Clone your forked repository to your local machine. the command should be something like: `git clone https://github.com/<YOUR_GITHUB_NAME>/Xpense.git`
+4. Open any terminal and navigate to the project directory.
+5. Run the `vagrant up` command.
+
```

Comment:
I think you should add:
6. Use any browser and navigate to - http://localhost:8000